packages/react-accordion/src/components/Accordion/useAccordionContext.ts

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react';
+import { createDescendantContext, useDescendant, useDescendantsInit } from '../../utils/descendants';
+import { AccordionContext, AccordionDescendant, AccordionIndex, AccordionState } from './Accordion.types';
+import { useConst, useControllableValue, useEventCallback } from '@fluentui/react-utilities';
+import { createContext } from '@fluentui/react-context-selector';
+
+export const accordionDescendantContext = createDescendantContext<AccordionDescendant>('AccordionDescendantContext');
+
+export const accordionContext = createContext<AccordionContext>({
+  openItems: [],
+  requestToggle() {
+    /* noop */
+  },
+});
+
+/**
+ * Creates the context to be provided for AccordionItem components
+ */
+export function useCreateAccordionContext(state: AccordionState) {
+  const { index, multiple, collapsible, onToggle, size, expandIcon, expandIconPosition, button } = state;
+  const isControlled = useConst(typeof index !== 'undefined');
+  const [descendants, setDescendants] = useDescendantsInit<AccordionDescendant>();
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const initialDefaultIndex = React.useMemo(() => initializeUncontrolledOpenItems(state), []);
+  const normalizedIndex = React.useMemo(() => (index !== undefined ? normalizeIndex(index) : undefined), [index]);
+  const [openItems, setOpenItems] = useControllableValue(normalizedIndex, initialDefaultIndex);
+
+  const requestToggle = useEventCallback((ev: React.MouseEvent<HTMLElement>, i: number) => {
+    onToggle?.(ev, i);
+    setOpenItems(previousOpenItems =>
+      updateOpenItems(i, previousOpenItems!, {
+        collapsible,
+        multiple,
+      }),
+    );
+  });
+  const context = {
+    openItems: isControlled ? normalizedIndex! : openItems!,
```

Comment:
```suggestion
    openItems,
```

IMO, it should just reuse `openItems` from `useControllableValue()`. Will something break if we will use like this?

Reply:
No. you're right, that might be a legacy from the implementation without `useControllableValue`